package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "fix-lockfile-integrity",
-    "version": "1.3.1",
+    "version": "1.3.2",
```

Comment:
No need for that. The publish process promotes the version number

Reply:
Gotcha, will revert that :)